tests/queries/0_stateless/01526_client_start_and_exit.sh

```diff
@@ -11,7 +11,7 @@ ${CLICKHOUSE_CLIENT} -q "SELECT 'CREATE TABLE test_' || hex(randomPrintableASCII
 function stress()
 {
     while true; do
-        "${CURDIR}"/01526_client_start_and_exit.expect-not-a-test-case | grep -v -P 'ClickHouse client|Connecting|Connected|:\) Bye\.|new year|^\s*$|spawn bash|^0\s*$'
+        "${CURDIR}"/01526_client_start_and_exit.expect-not-a-test-case | grep -v -P 'ClickHouse client|Connecting|Connected|:\) Bye\.|new year|^\s*$|spawn bash|\?2004l|^0\s*$'
```

Comment:
I didn't understand why this change checks correctness. Could you please explain it?

Reply:
It's most likely because parallel running `expect` emulated terminal doesn't work well with bracketed paste enabling sequence, which is `\e033?2004l`  (FYI https://cirw.in/blog/bracketed-paste)

Reply:
Then write a comment inside the test, otherwise it's very obscure.